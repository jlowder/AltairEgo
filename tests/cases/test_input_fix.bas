10 REM Test INPUT with comma-separated values
20 PRINT "Enter two numbers separated by comma: ";
30 INPUT A, B
40 PRINT "You entered: "; A; " and "; B
50 PRINT "Sum: "; A + B
60 END
