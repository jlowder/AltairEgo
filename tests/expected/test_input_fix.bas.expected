Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
Enter two numbers separated by comma: ? You entered:  0  and  0
Sum:  0
OK
